### DISCUSSION TITLE
Changing Instance Domain

### DISCUSSION THREAD

**Discussion by icecake0107** [misskey-dev/misskey#9254] [Q&A] [answered]
Hi dear Misskey friends,

I have a Misskey instance running but I would like to change the domain (from `foo.com` to `bar.io`).

I am not sure what the impact would be to the existing accounts, like for a user's Following & Followers/Avatar..., do they need to reconnect after the domain change? 🤔 If so how to do that? 

Thanks in advance! 😄

**Comment by ThatOneCalculator** [comment, marked as answer]
I would highly recommend ***not*** changing your URL. Due to how the Fediverse/ActivityPub works, changing your URL will most likely result in entirely broken federation.

**Reply by codingneko** [reply]
You did change yours tho, how'd you do it? I'm curious

**Reply by ThatOneCalculator** [reply]
1. I changed it with my servers running Firefish, not Misskey
2. The whole process involved multiple custom scripts talking both to the server API and the database directly and was overall extremely janky. I wouldn't recommend that anyone try and recreate what I did tbh.

**Reply by codingneko** [reply]
I honestly don't consider myself knowledgable enough to even attempt this, but it still amazes ms to this day you could do it. Also, can't remember this... you did the migration about 4 months ago right? do all Firefish accounts show up as 4 months old in remote instances? or were you able to make remote instances know the real creation dates of the accounts? 🤔

**Reply by ThatOneCalculator** [reply]
It was neither 4 months ago nor the real dates, because it reused data from calckey.social. The database from calckey.social and firefish.social are carbon-copies that were kept in-sync until the migration was complete, meaning no user data, remote or local, was ever altered in a meaningful way.
